src/set_theory/surreal/basic.lean

```diff
@@ -287,24 +321,14 @@ instance : ordered_add_comm_group surreal :=
   add_le_add_left   := by { rintros ⟨_⟩ ⟨_⟩ hx ⟨_⟩, exact pgame.add_le_add_left hx } }
 
 noncomputable instance : linear_ordered_add_comm_group surreal :=
-{ le_total := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical; exact
-    or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
+{ le_total     := by rintro ⟨⟨x, ox⟩⟩ ⟨⟨y, oy⟩⟩; classical;
+                     exact or_iff_not_imp_left.2 (λ h, le_of_lt oy ox (pgame.not_le.1 h)),
```

Comment:
What happened here? If you're changing anything here, it should be to remove the over-use of `;`, either using braces or begin ... end.

Reply:
I think these whitespace changes got tagged on due to some automatic editor alignment. I'll get rid of these.